### PR TITLE
Take changes from Monty PR

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

**Review by henkbirkholz — APPROVED** [review]
This PR adopts the changes from #20 (Monty's PR) and rebases it to the current head. Yogesh and I reveiwed every single change to the new base and it looks good! Ship it!

File: ietf-scitt-charter.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ Goals
 =====
 Based on an input document on the architecture (draft-birkholz-scitt-architecture-00), the WG will:
 
-1. Standardize the overall technical security flows for securing a software supply chain, which also includes firmware, and covering the essential building blocks that make up the architecture.
+1. 1. Standardize the technical flows for providing information about a software supply chain, which also includes firmware, and covering the essential building blocks that make up the architecture.
```
Comment (yogeshbdeshpande) [inline ietf-scitt-charter.md:30]: Noticed this bug, will correct it now!

Comment (henkbirkholz) [reply ietf-scitt-charter.md:30]: ack

**Review by henkbirkholz — APPROVED** [review]
numbering nit fixed

**Comment by yogeshbdeshpande** [comment]
This PR addresses https://github.com/ietf-scitt/charter/pull/20

**Pull request merged**